### PR TITLE
Feature/make match

### PR DESCRIPTION
**Pull request by csconner1998**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
+from tkinter.messagebox import NO
```
Comment (GNMoseke) [inline bot.py]: assuming another auto-import from vscode, can we remove

File: bot.py [hunk 2]
```diff
@@ -28,14 +28,15 @@
 
 TODO: manual game creation
 """
-db_handler = DatabaseHandler(host=os.environ.get('DB_HOST'), db_name=os.environ.get('DB_NAME'), user=os.environ.get('DB_USER'), password=os.environ.get('DB_PASS'))
+db_handler = DatabaseHandler(host=os.environ.get('DB_HOST'), db_name=os.environ.get(
+    'DB_NAME'), user=os.environ.get('DB_USER'), password=os.environ.get('DB_PASS'))
```
Comment (GNMoseke) [inline bot.py:30]: nit: What's up with the spacing on these lines? Might just be a github render thing

File: bot.py [hunk 3]
```diff
@@ -50,6 +51,7 @@
 main_leaderboard = None
 
 # Riot API watcher
+print(os.environ.get('Riot_Api_Key'))
```
Comment (GNMoseke) [inline bot.py]: nit: remove print unless necessary

File: bot.py [hunk 4]
```diff
@@ -347,6 +405,7 @@ async def handle_inhouse_role_reaction(payload: discord.RawReactionActionEvent):
         print(e)
         await payload.member.send("Check that your discord name has been linked to your summoner name in #name-assign correctly, then try again. If the problem persists, please contact a staff member.")
 
+
```
Comment (GNMoseke) [inline bot.py]: mega nit: extra whitespace

File: inhouse/command_handlers/match.py [hunk 5]
```diff
@@ -119,7 +120,7 @@ async def send_channel(self, member: discord.Member, channel: discord.VoiceChann
         except Exception as e:
             print(e)
             if member != None and channel != None:
```
Comment (GNMoseke) [inline inhouse/command_handlers/match.py]: I've figured out why this doesn't show a lot of the time: Occasionally, `member` will be `None`, so the move above will throw an exception. This join message then never gets returned because we can't pass the condition here, and if we did we'd get another exception from the `member.id` on line 123. I'm really not entirely sure why it fails sometimes, seems to be pretty inconsistent on who it gets as `None`. I thought it had to do with the "Member" role but seems to do it sometimes even if someone has it. 

TL;DR, this chunk of code will basically never do anything. Not necessary to change in this PR, but I think we should change it to just send the channels and not @ the missing members in the short term.

File: inhouse/command_handlers/test_player.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+class TestPlayer:
+  name = "Test"
```
Comment (GNMoseke) [inline inhouse/command_handlers/test_player.py]: Same comment as last PR on this, I think we can subclass this to Player (or remove if you merge it with that stuff)

**Review by GNMoseke — APPROVED** [review]
Another auto-import to remove but otherwise this LGTM

File: bot.py (hunk 3 above)
Comment (csconner1998) [reply bot.py]: Done

File: inhouse/command_handlers/match.py (hunk 5 above)
Comment (csconner1998) [reply inhouse/command_handlers/match.py]: Interesting... will maybe fix on bug fix branch and open PR

File: inhouse/command_handlers/test_player.py (hunk 6 above)
Comment (csconner1998) [reply inhouse/command_handlers/test_player.py]: Will rebase to main

File: bot.py (hunk 2 above)
Comment (csconner1998) [reply bot.py:30]: My VS code decided to auto format itself
